Src/VsVimShared/Implementation/Misc/StandardCommandTarget.cs

```diff
@@ -181,6 +194,55 @@ private bool TryProcessWithBuffer(KeyInput keyInput)
             return handled;
         }
 
+        /// <summary>
+        /// Try to process the key input with WPF
+        /// </summary>
+        /// <param name="keyInput"></param>
+        /// <returns></returns>
+        private bool TryProcessWithWpf(KeyInput keyInput)
+        {
+            var keyMap = new Dictionary<VimKey, Key>
+            {
+                { VimKey.Back, Key.Back },
+                { VimKey.Left, Key.Left },
+                { VimKey.Right, Key.Right },
+                { VimKey.Home, Key.Home },
+                { VimKey.End, Key.End },
+            };
+            foreach (var pair in keyMap)

```

Comment:
Rather than iterate here why not make the key of the dictionary be `KeyInput`? That would allow for a `TryGetValue` call.

Reply:
Yes, that makes sense. Will do.

Reply:
Fixed the dictionary key, made the dictionary static and added a couple more keys that were not being handled.